src/person.rb

```diff
@@ -1,14 +1,15 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
 
   def initialize(age, name = 'unknown', parent_permission: false)
     @id = rand(1...300)
     @age = age
     @name = name
     @parent_permission = parent_permission
+    @rentals = []
     super()
   end
```

Comment:
- [ ] [REQUIRED] Kindly add a  method for adding rentals.
Something like this ;
```ruby
 def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```

Reply:
Done.